extensions/ui/bit-start-cmd-output-templates/compilation-error-output-template.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from 'ink';
+import React from 'react';
+
+export type props = {
+  duration: string;
+  components: [any];
+};
+
+export const CompilationErrorOutput = ({ duration, components }: props) => {
+  const failedComponents = components.filter((cop) => cop.status === 'FAIL');
+  return (
+    <Box>
+      Compilation finished ({duration}) Failed to compile {failedComponents.length} components out of{' '}
+      {components.length} components STATUS COMPONENT ID
+      {components.map(
+        (component) => `${component.status === 'FAIL' ? '✗' : '✓'} ${component.status}        ${component.id}`
```

Comment:
this types of hard coded spaces after the status make it not aligned. use pad-left / pad-right we have already examples of this. Search for pad-right

Reply:
What do you mean by "alias member"?